spec/manageable_spec.rb

```diff
@@ -1,168 +1,185 @@
 require 'spec_helper'
 
+class InvalidSaveCat < ContentfulModel::Base
+  self.content_type_id = 'cat'
+
+  validate :always_fail, -> (e) { false }
+end
+
 describe ContentfulModel::Manageable do
-  before do
+  before :each do
     ContentfulModel.configure do |c|
       c.default_locale = 'en-US'
-      c.management_token = 'foo'
-      c.access_token = 'foobar'
-      c.space = 'cfexampleapi'
+      c.management_token = '<ACCESS_TOKEN>'
+      c.access_token = '4d0f55d940975f78139daae5d965b463c0816e88ad16062d2c1ee3d6cb930521'
```

Comment:
That's a CDA Token for my playground space, it's shared on purpose, CMA tokens are obfuscated.